modules/gitops-release-notes-1-7-0.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-7-0_{context}"]
+= Release notes for {gitops-title} 1.7.0
+
+{gitops-title} 1.7.0 is now available on {product-title} 4.8, 4.9, 4.10 and 4.11.
+
+[id="new-features-1-7-0_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, you can add environmental variables to the Notifications controller. link:https://issues.redhat.com/browse/GITOPS-2313[GITOPS-2313]
+
+* With this update, the default `nodeSelector "kubernetes.io/os": "linux"` will be added to all the workloads such that they only schedule on Linux nodes. Also any custom `nodeSelectors` are added to the default and take precedence if the key is same. link:https://issues.redhat.com/browse/GITOPS-2215[GITOPS-2215]
+
+* With this update, you can set custom `nodeSelectors` in the Operator workloads by editing their `GitopsService` custom resource. link:https://issues.redhat.com/browse/GITOPS-2164[GITOPS-2164]
+
+
+* With this update, you can choose the RBAC policy matcher mode. There are currently two options `glob` and `regex`. `Glob` is the default.link:https://issues.redhat.com/browse/GITOPS-1975[GITOPS-1975]
```

Comment:
Suggested version:

* With this update, you can use the RBAC policy matcher mode to select from the following options: `glob` (default) and `regex`.